dotcom-rendering/src/components/Metrics.importable.tsx

```diff
@@ -106,7 +157,15 @@ export const Metrics = ({ commercialMetricsEnabled }: Props) => {
 					),
 				);
 		},
-		[abTestApi, adBlockerInUse, commercialMetricsEnabled],
+		[
+			abTestApi,
+			adBlockerInUse,
+			browserId,
```

Comment:
Also FYI @arelra because perhaps the intricacies of this may crop up in other usages of `useOnce` 🤔

Reply:
Having worked on the interface of `initCommercialMetrics` and `initCoreWebVitals` I know believe it was an oversight to have allowed anything but a `string` to be passed to `browserId` and `pageViewId`, as we otherwise have no mechanism of joining the information in the data warehouse, making the collection of these values nearly pointless. I think that choosing explicit strings will make it easier to assess how much of our data falls into this category, so I’ve gone for the following states:

1. Pending – `undefined`
2. Failed to read cookie – `"no-browser-id-available" | "no-page-view-id-available"`
3. Got a valid cookie – `string`